app/code/Magento/Email/Model/Config/Source/SmtpAuthType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Email\Model\Config\Source;
+
+/**
+ * Option provider for custom media URL type
```

Comment:
Please double check your comments.  This description looks incorrect, as well as SmtpSslType's.